src/dotnet/APIView/APIViewWeb/Repositories/ReviewManager.cs

```diff
@@ -270,11 +270,13 @@ public async Task UpdateRevisionLabelAsync(ClaimsPrincipal user, string id, stri
 
         public async Task ToggleIsClosedAsync(ClaimsPrincipal user, string id)
         {
-            var review = await GetReviewAsync(user, id);
-            await AssertReviewOwnerAsync(user, review);
+            if (user == null)
```

Comment:
can this happen?

Reply:
What about?
```
            var result = await _authorizationService.AuthorizeAsync(user, reviewModel);
```

Reply:
Authorization needs an auth policy and it doesn't work just with context user and resource. I had added this check only to ensure this is not getting triggered as web request without login. This check is not required since GetReviewAsync ensures user is logged in.

Reply:
>This check is not required

Yeah, let's remove it then.